data/settings.json

```diff
@@ -1,12 +1,10 @@
 {
-  "default_raid_duration": 120,
-  "exclusive_raid_duration": 5760,
+  "standard_raid_incubate_duration": 60,
   "standard_raid_hatched_duration": 60,
+  "exclusive_raid_incubate_duration": 5760,
   "exclusive_raid_hatched_duration": 60,
   "start_clear_time": 10,
   "deletion_grace_time": 5,
-  "min_raid_hour": 6,
-  "max_raid_hour": 22,
```

Comment:
Does this mean raids can be created whenever now?

Reply:
Yes